disperser/apiserver/disperse_blob_v2.go

```diff
@@ -17,9 +17,27 @@ import (
 	"github.com/Layr-Labs/eigenda/encoding"
 	"github.com/Layr-Labs/eigenda/encoding/rs"
 	gethcommon "github.com/ethereum/go-ethereum/common"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 )
 
-func (s *DispersalServerV2) DisperseBlob(ctx context.Context, req *pb.DisperseBlobRequest) (*pb.DisperseBlobReply, error) {
+func (s *DispersalServerV2) DisperseBlob(
+	ctx context.Context,
+	req *pb.DisperseBlobRequest,
+) (*pb.DisperseBlobReply, error) {
+	reply, st := s.disperseBlob(ctx, req)
+	api.LogResponseStatus(s.logger, st)
+	if st != nil {
```

Comment:
Here you're checking whether the status is non-nil, and returning an error.

But looking at `disperseBlob`, the status is ALWAYS non-nil, even in the success case. Does this actually work?